scm/src/GFS_typedefs.F90

```diff
@@ -1493,10 +1495,6 @@ real (kind=kind_phys), pointer :: prsik (:,:) => null()   !< Exner function at i
 
     !--- MP quantities for 3D diagnositics 
     real (kind=kind_phys), pointer :: refl_10cm(:,:) => null()  !< instantaneous refl_10cm
-
-    !--- Extra PBL diagnostics
```

Comment:
Is this a new change? Or just something we missed in an earlier update?

Reply:
I think it was missed. Just an extra variable that wasn't hurting anything.